ftfy/cli.py

```diff
@@ -4,10 +4,30 @@
 Because files do not come with their encoding marked, it first runs the file
 through `ftfy.guess_bytes`, then runs it through `ftfy.fix_text`.
```

Comment:
This docstring is not fully accurate now that you're supposed to tell it the encoding.